utils/index.js

```diff
@@ -11,7 +11,7 @@ const DEFAULT_VERSION = {
   patch: 4,
 };
 
-exports.isEjected = fs.pathExistsSync(path.join(process.cwd(), 'config/webpack.config.dev.js'));
+exports.isEjected = fs.pathExistsSync(path.join(process.cwd(), 'config/paths.js'))
```

Comment:
paths.js isn't version dependent it seems.

Reply:
We could but I preferred to check for the webpack config as it is actually path we really need. Anyway that's ok.